Cargo.lock

```diff
@@ -303,7 +303,7 @@ checksum = "e78d4f1cc4ae33bbfc157ed5d5a5ef3bc29227303d595861deb238fcec4e9457"
 
```

Comment:
Would be nice to follow [the 50/72 rule](https://www.midori-global.com/blog/2018/04/02/git-50-72-rule) for commit messages. Also please add a `Signed-off-by` line: `git commit -s --amend`.

Reply:
I'll rebase on `main` and update the commit message.